module1/homework/grayscaleImages/test.cpp

```diff
@@ -3,7 +3,7 @@
 #include <utility>  // for std::pair<>
 #include <vector>
 
-// TODO: include
+#include "compression.hpp"
```

Comment:
Vector is included in the header,so you could remove it here.

Reply:
Ok thanks, oversight